app/views/incomeSources/cease/CeaseIncomeSources.scala.html

```diff
@@ -113,32 +109,30 @@
         @p(id = Some("uk-property-p1")) {
             @getMessage("uk-property.p1") }
 
-        @govukTable(Table(
-            head = Some(List(
-                HeadCell(
-                    content = Text(getMessage("table-head.date-started")),
-                    attributes = Map("id" -> "table-head-date-started-uk")),
-                HeadCell(
-                    content = Empty,
-                    attributes = Map("id" -> "empty2"))
-            )),
-
-            rows = Seq(Seq(
-                TableRow(
-                    content = Text(sources.ukProperty.map(_.tradingStartDate.toLongDate).get),
-                    attributes = Map("id" -> "table-row-trading-start-date-uk")),
-                TableRow(
-                    content = HtmlContent(
-                        s"""<dd class="govuk-summary-list__actions">
-                                <a class="govuk-link" href='$ceaseUkPropertyURL'>
-                                    ${getMessage("cease")}<span class="govuk-visually-hidden">cease</span>
-                                </a>
-                            </dd>"""
-                    )
-                )
-            )),
-            attributes = Map("id" -> "uk-property-table")
-        ))
+        <table class="govuk-table govuk-!-margin-bottom-3" id="uk-property-table">
+            <thead class="govuk-table__head">
+                <tr class="govuk-table__row">
+                    <th class="govuk-table__header" scope="col" id="table-head-date-started-uk">@getMessage("table-head.date-started")</th>
+                    <td class="govuk-table__header" id="empty1"></td>
+                </tr>
+            </thead>
+            <tbody>
+                <tr class="govuk-table__row">
+                    <td class="govuk-table__cell" id="table-row-trading-start-date-uk">@sources.ukProperty.map(_.tradingStartDate.toLongDate).get</td>
```

Comment:
the whole point to have a separate model was to avoid these hidden bugs, essentially you not handling situation when ukProperty field is not set;

Reply:
@if(sources.ukProperty.nonEmpty) {
this line at the top of the block solves this i think. ukProperty field is an option so the table only shows when the option is Some(ukProperty)